### DISCUSSION TITLE
Problems with the pluging

### DISCUSSION THREAD

**Discussion by joselr04** [brianpetro/obsidian-smart-connections#613] [General]
I've started testing the plugin and I'm having a lot of issues. I'm using Meta Llama 3 Instruct 7B Q4 in LLM studio.

Looking at the LLM studio log, some very strange things are happening:

My input: Summary of my note [[xxxxxxx]]

This is received by the LLM, which apparently, according to the log, invents a response, since the note was never passed to it at any time.

Immediately afterwards, there is another call. Another message, with a different system prompt than the previous one. Here, the content of the notes is present, but not the note that I wanted.

Often, it sends several things in 2 consecutive and different messages.

The truth is that I'm getting very bad results. I generally don't get answers in the chat, many times the model tells me it doesn't have access to the information. The text I write in the chat disappears. Is there something I'm doing wrong?

**Comment by christianngarcia** [comment]
Including a self-referential pronoun, in this case "my," then note lookup/retrieval is being triggered and that interferes with the specifying the note as context 🌼
